modules/gitops-release-notes-1-10-0.adoc

```diff
@@ -0,0 +1,118 @@
+** `active_argocd_instances_total` - The number of Argo CD instances currently being managed across the cluster.
+** `active_argocd_instances_by_phase{phase="<_PHASE>"}` - The number of managed Argo CD instances in the given phase.
+** `active_argocd_instance_reconciliation_count{namespace="<_YOUR-DEFINED-NAMESPACE>"}` - The number of times the instance in a given namespace has been reconciled.
+** `controller_runtime_reconcile_time_seconds_per_instance{namespace="<_YOUR-DEFINED-NAMESPACE>"}` - The distribution of reconciliation cycles by their durations for the instance in a given namespace. link:https://issues.redhat.com/browse/GITOPS-3071[GITOPS-3071]
+
+* With this update, the Argo CD CRD API version has been upgraded from v1alpha1 to v1beta1 in order to accomodate the breaking changes resulting from the deprecation of `.spec.dex` and certain `.spec.sso` fields. To streamline the automatic migration of existing v1alpha1 ArgoCD CRs to v1beta1, conversion webhook support has been implemented. link:https://issues.redhat.com/browse/GITOPS-3040[GITOPS-3040]
++
+[NOTE]
+====
+By default, the conversion webhook is enabled only for OLM-installed operators. For non-OLM installations of the operator, enabling the webhook is optional. However, without conversion webhook support, you are responsible for manually migrating any existing ArgoCD v1alpha1 CRs to v1beta1.
+====
+
+
+[id="deprecated-features-1-10-0_{context}"]
+== Deprecated and removed features
+
+* With this release, the following deprecated `sso` and `dex` fields have been removed from ArgoCD CR:
+** The `.spec.sso.image`, `.spec.sso.version`, `.spec.sso.resources`, and `.spec.sso.verifyTLS` fields for keycloak SSO configurations.
+** The `.spec.dex` fields, along with `DISABLE_DEX` environment variable.
++
+Additionally, the `.status.dex` and `.status.ssoConfig` fields have also been removed, and a new status field, `.status.sso`, has been introduced.The new field reflects the workload status of the SSO provider (dex or keycloak) configured via the `.spec.sso.provider` field. link:https://issues.redhat.com/browse/GITOPS-2473[GITOPS-2473]
++
+[IMPORTANT]
+====
+To configure dex or keycloak SSO, use the equivalent fields under `.spec.sso`.
+==== 
+
+* With this update, the deprecated `.spec.resourceCustomizations` field has been removed from ArgoCD CR. Bug fixes and support will only be provided through the end of the {gitops-title} v1.9 lifecycle.
+As an alternative to `.spec.resourceCustomizations`, you can use `.spec.resourceHealthChecks`, `.spec.resourceIgnoreDifferences`, and `.spec.resourceActions` fields instead. link:https://issues.redhat.com/browse/GITOPS-3041[GITOPS-3041]
++
+[IMPORTANT]
+====
+To prevent data loss during upgrade to {gitops-title} Operator v1.10.0, ensure that you backup `.spec.resourceCustomization` value if it is used in your ArgoCD CRs.
+==== 
```

Comment:
```suggestion
* With this update, the deprecated `.spec.resourceCustomizations` field is removed from Argo CD CR. Bug fixes and support are only provided through the end of the {gitops-title} v1.9 lifecycle.
As an alternative to `.spec.resourceCustomizations`, you can use `.spec.resourceHealthChecks`, `.spec.resourceIgnoreDifferences`, and `.spec.resourceActions` fields instead. link:https://issues.redhat.com/browse/GITOPS-3041[GITOPS-3041]
+
[IMPORTANT]
====
To prevent data loss during upgrade to {gitops-title} Operator v1.10.0, ensure that you backup `.spec.resourceCustomization` value if it is used in your Argo CD CRs.
==== 
```